TikTokApi/browser_utilities/browser.py

```diff
@@ -248,7 +251,7 @@ def clean_up(self):
         try:
             self.browser.close()
         except Exception:
-            logging.info("cleanup failed")
+            log.exception("cleanup failed")
```

Comment:
added exc stack trace & body with `log.exception` which includes `exc_info=True` to provide more debugging info